Brand kit updates

AKCodez opened the pull request.

AKCodez committed:
moves brand kit link in footer and nav to resources, adds new brand kit deletes old one replaces all references to new url

AKCodez committed:
moves brand kit link in footer and nav to resources, adds new brand kit deletes old one replaces all references to new url

amarantha-k reviewed (commented):
The local build looked good. 
Wondering if the zip files should be under `assets` instead of the `content` folder? @mDuo13 what do you think?

AKCodez commented:
> The local build looked good. Wondering if the zip files should be under `assets` instead of the `content` folder? @mDuo13 what do you think?

I put it in content as that is where it was being hosted before.

mDuo13 commented on content/XRPL_Brand_Kit.zip:
I don't love that this zip has a `__MACOSX/` folder at the top and `.DS_Store` files in every dir, but it's fine I guess.

amarantha-k replied:
While not a blocker, I'd prefer that we remove these files as they sometimes can cause problems.

mDuo13 reviewed (approved):
It's OK for the brand kit to be in `content/` because we're linking people to download it from GitHub via the `github.com/.../raw/...` link. However, if we moved it under `assets/`, then we could serve it directly from xrpl.org at a URL like `https://xrpl.org/assets/XRPL_Brand_Kit.zip`, which _might_ make it easier to track how many times it's been downloaded (using the server access log or something).

tequdev commented on dactyl-config.yml:
```diff
@@ -4945,6 +4945,21 @@ pages:
+        html: https://github.com/XRPLF/xrpl-dev-portal/raw/master/content/XRPL_Brand_Kit.zip
+        parent: resources.html
+        blurb: XRPLのデザインアセットのガイドラインを確認しましょう。
+        top_nav_grouping: Development
```
```suggestion
        top_nav_grouping: 開発
```

In `targets: ja`, "開発" is used for “Development"

https://github.com/XRPLF/xrpl-dev-portal/blob/db87dc69635b18c0d16bb8eab680a3b86258599b/dactyl-config.yml#L4788

AKCodez committed:
Update dactyl-config.yml

Co-authored-by: tequ <git@tequ.dev>

AKCodez committed:
moves zip to assets folder

AKCodez commented:
> It's OK for the brand kit to be in `content/` because we're linking people to download it from GitHub via the `github.com/.../raw/...` link. However, if we moved it under `assets/`, then we could serve it directly from xrpl.org at a URL like `https://xrpl.org/assets/XRPL_Brand_Kit.zip`, which _might_ make it easier to track how many times it's been downloaded (using the server access log or something).

Moved into assets
